docker-compose.yml

```diff
@@ -1,21 +1,24 @@
 version: "3"
 services:
   hapi-fhir-jpaserver-start:
+    network_mode: host
     build: .
     container_name: hapi-fhir-jpaserver-start
     restart: on-failure
     ports:
       - "8080:8080"
-  hapi-fhir-mysql:
-    image: mysql:latest
-    container_name: hapi-fhir-mysql
-    restart: always
     environment:
-      MYSQL_DATABASE: 'hapi'
-      MYSQL_USER: 'admin'
-      MYSQL_PASSWORD: 'admin'
-      MYSQL_ROOT_PASSWORD: 'admin'
-    volumes:
-      - hapi-fhir-mysql:/var/lib/mysql
-volumes:
-  hapi-fhir-mysql:
\ No newline at end of file
+      JAVA_OPTS: "-Dhapi.properties=/usr/local/tomcat/webapps/ROOT/WEB-INF/classes/hapi-r4.properties -Denable-web=false -Durl_pattern=/fhir/*"
+      HAPI_FHIR_VERSION: "R4"
+      HAPI_SERVER_ADDRESS: "https://<fhirDomain>/fhir/"
+      HAPI_DATASOURCE_DRIVER: "com.mysql.jdbc.Driver"
+      HAPI_DATASOURCE_URL: "jdbc:mysql://<mysqlHost>:<mysqlHostPort>/<dbName>"
+      HAPI_DATASOURCE_USERNAME: "<dbUserName>"
+      HAPI_DATASOURCE_PASSWORD: "<dbPassword>"
+      HAPI_HIBERNATE_DIALECT: "org.hibernate.dialect.MySQL5InnoDBDialect"
+      HAPI_SUBSCRIPTION_RESTHOOK_ENABLED: "true"
+      HAPI_SUBSCRIPTION_WEBSOCKET_ENABLED: "true"
+      API_KEY: "<apiKey>"
+      VALIDATE_API_KEY: "true"
```

Comment:
This value should be `false` in here. For local development and testing, the default is `false`.

Reply:
ok